hs_core/views/__init__.py

```diff
@@ -933,7 +941,7 @@ def _share_resource(request, shortkey, privilege, user_or_group_id, user_or_grou
                               'error_msg': err_message}
 
     else:
-        group_pic_url = 'No picture provided'
+        group_pic_url = None
         if group_to_share_with.gaccess.picture:
```

Comment:
did you test with and without to ensure it is ok with the None type instead of str?

Reply:
Yes, I tested this both with and without a picture.